src/pages/trade/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { useEffect } from "react";
+import Router from "next/router";
+
+const IndexPage = () => {
+  useEffect(() => {
+    const { pathname } = Router;
+    if (pathname == "/trade") {
+      Router.push("/trade/SOL-USD");
```

Comment:
why is this necessary? what was the bug with defaulting the tokens before?

Reply:
was using dydx and binance navigation system instead of gmx, i think its a cleaner trading ux